beangulp/importers/csv_test.py

```diff
@@ -40,6 +40,14 @@ def test_normalize_config__with_header(self):
         iconfig, _ = csv.normalize_config({Col.DATE: 1}, head)
         self.assertEqual({Col.DATE: 1}, iconfig)
 
+    def test_normalize_config__has_header_false(self):
+        head = textwrap.dedent("""\
+          DEBIT,3/18/2016,"Payment to Chafe card ending in 1234 03/18",-2680.89,ACCT_XFER,3409.86,,
+        """)
+        iconfig, has_header = csv.normalize_config({Col.DATE: 1}, head,
+                                                   skip_lines=4, has_header=False)
```

Comment:
You are skipping 4 lines in a file that has only one. I don't think this is testing what you think ti is testing.